src/client/browser/idle-page/index.html.mustache

```diff
@@ -2,13 +2,114 @@
 <html lang="en">
 <head>
     <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge"/>
     <title></title>
-    <script type="application/x-javascript" src="/browser/assets/index.js"></script>
-    <script type="text/javascript">
-        init('{{{statusUrl}}}');
-    </script>
+    <script type="text/javascript" src="/browser/assets/index.js"></script>
+    <link rel="stylesheet" href="/browser/assets/styles.css"/>
+    <link rel="icon" type="image/x-icon" href="/favicon.png"/>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no">
 </head>
 <body>
-{{{userAgent}}}
+<div class="page-background">
+    <div class="container">
+        <canvas class="spinner"></canvas>
+        <div class="status-text-area">
+                <svg class="logo" version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
```

Comment:
Can we save logo to `svg` file and fetch and insert it as resource? This inline SVG code looks scary :fearful: